nifi-nar-bundles/nifi-iceberg-bundle/nifi-iceberg-common/src/main/java/org/apache/nifi/processors/iceberg/AbstractIcebergProcessor.java

```diff
@@ -115,18 +116,26 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
                 });
 
             } catch (Exception e) {
-                getLogger().error("Privileged action failed with kerberos user " + kerberosUser, e);
-                session.transfer(session.penalize(flowFile), REL_FAILURE);
+                if (!handleAuthErrors(e, session, context)) {
+                    getLogger().error("Privileged action failed with kerberos user " + kerberosUser, e);
```

Comment:
Use interpolation and not concatenation.
```suggestion
                    getLogger().error("Privileged action failed with kerberos user {}" , kerberosUser, e);
```